### DISCUSSION TITLE
Uvicron restapi with FastAPI shows shutting down while inferencing using pytorch models

### DISCUSSION THREAD

**Discussion by pradeepdev-1995** [fastapi/fastapi#8871] [Questions] [answered]
I tried to load and inference from a pytorch model using fastAPI(with uvicorn).
But while predicting time it shows like this

INFO:     Shutting down
INFO:     Waiting for connections to close. (CTRL+C to force quit)

I tried the same method using flask and worked fine within milliseconds.                    
Here is the code for  pretrained  model prediction using fastapi

![Screenshot from 2021-05-24 10-28-39](https://user-images.githubusercontent.com/41164884/119298598-fa853d00-bc7a-11eb-8974-5da96f096ee8.png)

But While predicting with fastAPI it shows like this

![Screenshot from 2021-05-24 09-53-15](https://user-images.githubusercontent.com/41164884/119296447-92ccf300-bc76-11eb-813b-6e64b2996d72.png)

(Since it is a pytorch RoBERTA based huge pretrained model,  the model size is around 600MB)

**Comment by Kludex** [comment]
Please follow our issue template: https://github.com/tiangolo/fastapi/issues/new/choose

**Comment by pradeepdev-1995** [comment]
@Kludex updated
Please put your thoughts in https://github.com/tiangolo/fastapi/issues/3283

**Comment by surry** [comment, marked as answer]
Looks like this and #3283 were solved by the suggestion here: https://github.com/tiangolo/fastapi/issues/3283#issuecomment-851002452

Can this and #3283 be closed @pradeepdev-1995?
